frontend/src/store/playbackManager.ts

```diff
@@ -447,7 +449,7 @@ class PlaybackManagerStore {
         seekto: (action): void => {
           this.currentTime = action.seekTime || 1;
         }
-      } as { [key in MediaSessionAction]?: MediaSessionActionHandler };
+      };
```

Comment:
As said above, any linter rule to have this enforced by default is more than appreciated!